fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -648,40 +694,179 @@ public void testCreateVersionOfBinaryWithBody() throws Exception {
+
+        final String mementoUri = createContainerMementoWithBody(descriptionUri, null);
+        assertMementoUri(mementoUri, descriptionUri);
+
+        setDescriptionProperty(id, null, DC.title.getURI(), "Updated");
+
+        try (final CloseableDataset dataset = getDataset(new HttpGet(mementoUri))) {
+            final DatasetGraph results = dataset.asDatasetGraph();
+
+            final Node mementoSubject = createURI(subjectUri);
+
+            assertFalse("Property added to original must not appear in memento",
+                    results.contains(ANY, mementoSubject, DC.title.asNode(), ANY));
+            assertFalse("Memento type should not be visible",
+                    results.contains(ANY, mementoSubject, RDF.type.asNode(), MEMENTO_TYPE_NODE));
+            assertFalse("Property added to original must not appear in memento",
+                    results.contains(ANY, mementoSubject, DC.title.asNode(), ANY));
+        }
+
+        // No binary memento should be created when specifically creating a description memento.
+        final String hypotheticalBinaryUri = mementoUri.replaceAll("fcr:metadata/fcr:versions", "fcr:versions");
+        assertEquals(NOT_FOUND.getStatusCode(), getStatus(new HttpGet(hypotheticalBinaryUri)));
+    }
+
+    /*
+     * Attempt to create binary description with container triples
+     */
+    @Test
+    public void testCreateVersionOfBinaryDescriptionInvalidTriples() throws Exception {
+        final String containerId = getRandomUniqueId();
+        final String containerSubjectUri = serverAddress + containerId;
+        createObjectAndClose(containerId);
+
+        createVersionedBinary(id);
+
+        final String descriptionUri = subjectUri + "/fcr:metadata";
+
+        final String containerBody = createContainerMementoBodyContent(containerSubjectUri, "text/n3");
+        final HttpPost createMethod = postObjMethod(descriptionUri);
+        createMethod.addHeader(CONTENT_TYPE, "text/n3");
+        createMethod.setEntity(new StringEntity(containerBody));
+
+        // Attempt to create memento with partial record
+        try (final CloseableHttpResponse response = execute(createMethod)) {
+            assertEquals("Didn't get a BAD_REQUEST response!", BAD_REQUEST.getStatusCode(), getStatus(response));
+        }
+    }
+
+    @Test
+    public void testCreateVersionBinaryDescriptionWithBodyAndDatetime() throws Exception {
+        createVersionedBinary(id);
+
+        final String descriptionUri = subjectUri + "/fcr:metadata";
+
+        final String mementoUri = createContainerMementoWithBody(descriptionUri, MEMENTO_DATETIME);
+        assertMementoUri(mementoUri, descriptionUri);
+
+        try (final CloseableDataset dataset = getDataset(new HttpGet(mementoUri))) {
+            final DatasetGraph results = dataset.asDatasetGraph();
 
-        createObject(pid);
+            final Node mementoSubject = createURI(mementoUri);
+
+            assertFalse("Memento type should not be visible",
```

Comment:
Is there also an `assertTrue` test that we can include that proves that there is a property that we expect to find?